plugins/outputs/sql/sql_test.go

```diff
@@ -353,7 +353,6 @@ func TestClickHouseIntegration(t *testing.T) {
 		WaitingFor: wait.ForAll(
 			wait.NewHTTPStrategy("/").WithPort(nat.Port("8123")),
 			wait.ForListeningPort(nat.Port(servicePort)),
-			wait.ForLog("Saved preprocessed configuration to '/var/lib/clickhouse/preprocessed_configs/users.xml'").WithOccurrence(2),
```

Comment:
As you said in the other PR, waiting for the log is not absolutely necessary. However, I would really feel safer if we would _also_ wait for the `Ready for connections.` log...